apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/Trace/TraceTest.cs

```diff
@@ -138,7 +139,8 @@ private void BlockUntilClockTick()
         public async Task Trace_Simple_Buffer()
         {
             string rootSpanName = CreateRootSpanName(nameof(Trace_Simple_Buffer));
-            BufferingTraceConsumer consumer = BufferingTraceConsumer.Create(CreateGrpcTraceConsumer());
+            SizedBufferingConsumer<TraceProto> consumer = SizedBufferingConsumer<TraceProto>.Create(
```

Comment:
Done, replaced with var in many places. 

I just avoid var as it reminds me too much of JavaScript... :)